packages/elements/src/chart/index.ts

```diff
@@ -160,6 +264,157 @@ export class Chart extends BasicElement {
     }
   }
 
+  /**
+   * Get as CSS variable and tries to convert it into a usable number
+   * @returns {(number|undefined)} The value as a number, or, undefined if NaN.
+   */
+  protected cssVarAsNumber (...args: string[]): number | undefined {
```

Comment:
It can't move to helper because this method use getComputedVariable. And getComputedVariable method can access only within a Class. 
```typescript
Property 'getComputedVariable' is protected and only accessible within class 'BasicElement' and its subclasses.ts(2445)
```

Reply:
Oh I missed that one. Marking as resolved.